src/drivers/dshot/module.yaml

```diff
@@ -40,6 +40,36 @@ parameters:
             decimal: 2
             increment: 0.01
             default: 0.055
+        DSHOT_3D_ENABLE:
+            description:
+                short: Allows for 3d mode when using DShot and suitable mixer
+                long: |
+                    This splits the throttle ranges in two.
```

Comment:
Can you also mention that the ESC's need to be configured to be in 3D mode?

Reply:
Done, and I also added that DSHOT_MIN should be set to 0 because it adds an unnecessary offset